cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/Person.java

```diff
@@ -0,0 +1,200 @@
+package no.unit.nva.cristin.person.model.nva;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+import static no.unit.nva.cristin.person.Constants.PERSON_CONTEXT;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.net.URI;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+@JsonPropertyOrder({"@context"})
+public class Person {
+
+    @JsonInclude(NON_NULL)
+    @JsonProperty("@context")
+    private static String context = PERSON_CONTEXT;
+    private URI id;
+    private List<TypedValue> identifiers;
+    private List<TypedValue> names;
+    private ContactDetails contactDetails;
+    private URI image;
+    private List<Affiliation> affiliations;
+
+    public Person() {
+
+    }
+
+    /**
+     * Creates a Person for serialization to client.
+     *
+     * @param id             Identifier of Person.
+     * @param identifiers    Different identifiers related to this object.
+     * @param names          Different names for this Person.
+     * @param contactDetails How to contact this Person.
+     * @param image          URI to picture of this Person.
+     * @param affiliations   This person's organization affiliations.
+     */
+    @JsonCreator
+    public Person(@JsonProperty("id") URI id, @JsonProperty("identifiers") List<TypedValue> identifiers,
+                  @JsonProperty("names") List<TypedValue> names,
+                  @JsonProperty("contactDetails") ContactDetails contactDetails, @JsonProperty("image") URI image,
+                  @JsonProperty("affiliations") List<Affiliation> affiliations) {
+        this.id = id;
+        this.identifiers = identifiers;
+        this.names = names;
+        this.contactDetails = contactDetails;
+        this.image = image;
+        this.affiliations = affiliations;
+    }
+
+    public String getContext() {
+        return context;
+    }
+
+    public List<TypedValue> getIdentifiers() {
+        return Objects.nonNull(identifiers) ? identifiers : Collections.emptyList();
+    }
+
+    public URI getId() {
+        return id;
+    }
+
+    public void setIdentifiers(List<TypedValue> identifiers) {
+        this.identifiers = identifiers;
+    }
+
+    public List<Affiliation> getAffiliations() {
+        return Objects.nonNull(affiliations) ? affiliations : Collections.emptyList();
+    }
+
+    public ContactDetails getContactDetails() {
+        return contactDetails;
+    }
+
+    public URI getImage() {
+        return image;
+    }
+
+    public void setContext(String thatContext) {
```

Comment:
Is context not static?

Reply:
Yes but i want the option to exclude it if we use it in a SearchResponse

Reply:
Do you have any suggestions to how i should make context optional? I want to use it in lookup but not in query.